### PR TITLE
(BIDS-2142) fix event value parsing

### PR DESCRIPTION
**Pull request by MauserBitfly**
### <samp>🤖 Generated by Copilot at de48be7</samp>

Fix eth1 transaction value display for non-string types. Use `fmt.Sprintf` to convert values with non-string type prefixes in `eth1data/eth1data.go`.

### CODE REVIEW

File: eth1data/eth1data.go
Comment (Eisei24) [inline eth1data/eth1data.go:1]: **Suggestion:** Can't we just replace the `%s` with an `%v` in line 211 instead?

**Review by Eisei24 — CHANGES_REQUESTED** [review]
Please have a look at my suggestion.

File: eth1data/eth1data.go
Comment (MauserBitfly) [reply eth1data/eth1data.go:1]: You are right, that's easier and I did not find a case where it has a bad effect

**Review by Eisei24 — APPROVED** [review]
lgtm

**Pull request merged**